test/OpenTelemetry.Exporter.Prometheus.HttpListener.Tests/PrometheusCollectionManagerTests.cs

```diff
@@ -40,9 +40,9 @@ public async Task EnterExitCollectTest()
 
             using (var provider = Sdk.CreateMeterProviderBuilder()
                 .AddMeter(meter.Name)
-#if NETFRAMEWORK
+#if PROMETHEUS_HTTP_LISTENER
                 .AddPrometheusHttpListener()
-#else
+#elif PROMETHEUS_ASPNETCORE
                 .AddPrometheusExporter()
 #endif
```

Comment:
@reyang , What do you think about following code? If you are fine, please commit suggestion to the code.
```suggestion
#else
#error Add PrometheusExporter to the MeterProvider.
#endif
```